src/applications/vaos/containers/ReasonForAppointmentPage.jsx

```diff
@@ -11,51 +11,92 @@ import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
 import { getFormPageInfo } from '../utils/selectors';
 import { scrollAndFocus } from '../utils/scrollAndFocus';
-import { PURPOSE_TEXT } from '../utils/constants';
+import { PURPOSE_TEXT, FACILITY_TYPES } from '../utils/constants';
 import TextareaWidget from '../components/TextareaWidget';
 import { validateWhiteSpace } from 'platform/forms/validations';
 
 const initialSchema = {
-  type: 'object',
-  required: ['reasonForAppointment', 'reasonAdditionalInfo'],
-  properties: {
-    reasonForAppointment: {
-      type: 'string',
-      enum: PURPOSE_TEXT.map(purpose => purpose.id),
-      enumNames: PURPOSE_TEXT.map(purpose => purpose.label),
+  default: {
+    type: 'object',
+    required: ['reasonForAppointment', 'reasonAdditionalInfo'],
+    properties: {
+      reasonForAppointment: {
+        type: 'string',
+        enum: PURPOSE_TEXT.map(purpose => purpose.id),
+        enumNames: PURPOSE_TEXT.map(purpose => purpose.label),
+      },
+      reasonAdditionalInfo: {
+        type: 'string',
+      },
     },
-    reasonAdditionalInfo: {
-      type: 'string',
+  },
+  cc: {
+    type: 'object',
+    properties: {
+      reasonAdditionalInfo: {
+        type: 'string',
+      },
     },
   },
 };
 
 const uiSchema = {
-  reasonForAppointment: {
-    'ui:widget': 'radio',
-    'ui:title': 'Please let us know why you’re making this appointment.',
+  default: {
+    reasonForAppointment: {
+      'ui:widget': 'radio',
+      'ui:title': 'Please let us know why you’re making this appointment.',
+    },
+    reasonAdditionalInfo: {
+      'ui:widget': TextareaWidget,
+      'ui:options': {
+        rows: 5,
+        expandUnder: 'reasonForAppointment',
+        expandUnderCondition: reasonForAppointment => !!reasonForAppointment,
+      },
+      'ui:validations': [validateWhiteSpace],
+    },
   },
-  reasonAdditionalInfo: {
-    'ui:widget': TextareaWidget,
-    'ui:options': {
-      rows: 5,
-      expandUnder: 'reasonForAppointment',
-      expandUnderCondition: reasonForAppointment => !!reasonForAppointment,
+  cc: {
+    reasonAdditionalInfo: {
+      'ui:widget': TextareaWidget,
+      'ui:title':
+        'Please let us know any additional details about your symptoms that may be helpful for the community health provider to know. (Optional)',
+      'ui:options': {
+        rows: 5,
+      },
+      'ui:validations': [validateWhiteSpace],
     },
-    'ui:validations': [validateWhiteSpace],
   },
 };
 
 const pageKey = 'reasonForAppointment';
-const pageTitle = 'Choose a reason for your appointment';
 
 export class ReasonForAppointmentPage extends React.Component {
   componentDidMount() {
-    document.title = `${pageTitle} | Veterans Affairs`;
-    this.props.openReasonForAppointment(pageKey, uiSchema, initialSchema);
+    const isCommunityCare = this.isCommunityCare();
+    const reasonUiSchema = isCommunityCare ? uiSchema.cc : uiSchema.default;
+    const reasonInitialSchema = isCommunityCare
```

Comment:
You could choose the schema in a constructor, that would avoid some of the checking you have to do in the `render` method.